data/retrieval_evaluator/eval_readme.md

```diff
@@ -0,0 +1,37 @@
+# RetrievalEvaluator Usage Guide
+The RetrievalEvaluator is a tool for capturing and analyzing memory retrieval data. Here's a quick guide on how to use it in your project:
+## 1. Initialization
+First, import and create an instance of the RetrievalEvaluator:
+```
+from retrieval_evaluator.retrieval_evaluator import RetrievalEvaluator
+
+evaluator = RetrievalEvaluator()
+```
+
+## 2. Capturing Comparison Data
+During your benchmark run, capture data for each query:
+```
+query = "Your query here"
+relevant_memories = [...]  # List of retrieved memories
+filtered_memories = [...]  # List of filtered memories
+
+evaluator.capture_comparison_data(query, relevant_memories, filtered_memories)
+```
+Note: You don't need to specify the benchmark version at this stage.
+
+## 3. Generating Output
+After your benchmark run is complete, generate the output for a specific benchmark version:
+   `evaluator.output("Dev Benchmark 1 32k - 2 Examples")`
+
+## 4. Output Location
+The evaluation results will be saved in:
+
+Plots and result files: retrieval_evaluator/evaluation_outputs/evaluation_{benchmark_version}_{timestamp}/
```

Comment:
Does this still work this way? I would expect it to require a file path to save to and, if not provided, just show it on screen.